src/styles/variables.scss

```diff
@@ -0,0 +1,7 @@
+$watch-size: 80vmin;
+$sec-size: 38vmin;
+$min-size: 20vmin;
+$center-radius: 5vmin;
+$basic-time: 10s;
+$seconds-coefficient: 60;
+$speed-up-time: 10s;
```

Comment:
Similarly, for `$speed-up-time`, consider renaming it to reflect its purpose, e.g., `$animation-duration-speed-up`, for clarity and consistency with the code style rules.